repose-aggregator/docs/src/asciibinder/architecture/system-model.adoc

```diff
@@ -170,6 +170,44 @@ If set to localhost, any physical node will match and run the *Repose* node.
      In this case, no services are being used.
 <13> Defines the required one default endpoint.
 
+==== Streaming Request Body to the Origin Service
+Streaming a request body to the origin service is accomplished by using the `chunked` transfer encoding.
+
+[source,xml]
+.partial http-connection-pool.cfg
+----
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+  <repose-cluster id="repose_cluster1"
+                  chunked-encoding="auto"> <!--1-->
+
+    <nodes>
+      <node id="repose_node" hostname="localhost" http-port="8080"/>
+    </nodes>
+
+    <filters/>
+
+    <destinations>
+      <endpoint id="origin_service" protocol="http" port="80" default="true"/>
+    </destinations>
+  </repose-cluster>
+</system-model>
+----
+<1> Determines whether or not the request body should be `chunked` when forwarded to the origin service.
```

Comment:
I thought we decided to put this on the destinations? In fact i thought i saw that in the code? Forget to update the docs?

Reply:
Dang it. Yes, I just forgot to update the docs. Will fix.